sys/compat/freebsd64/freebsd64_process.c

```diff
@@ -99,8 +99,8 @@ freebsd64_ptrace(struct thread *td, struct freebsd64_ptrace_args *uap)
 		struct dbreg dbreg;
 		struct fpreg fpreg;
 		struct reg reg;
-		char args[sizeof(td->td_sa.args)];
-		struct ptrace_sc_ret *psr;
```

Comment:
The '*psr' -> 'psr' bit is a mis-merge that could be a separate commit if really desired.